regen-network/regen-web
# chore: APP-40 social icons updates

## Description

https://regennetwork.atlassian.net/browse/APP-40?atlOrigin=eyJpIjoiMzZmNWFiNDRhOWE3NDI0NjlmMTNmYzBhYmNmM2MzZDUiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. Check out footer both on https://deploy-preview-2349--regen-marketplace.netlify.app/ and https://deploy-preview-2349--regen-website.netlify.app/
2. From https://deploy-preview-2349--regen-marketplace.netlify.app/, log in and add a twitter link to your profile and check the X icon is being used on your profile page, as on https://deploy-preview-2349--regen-marketplace.netlify.app/profiles/regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46/portfolio
3. Check out "Connect with us" section on https://deploy-preview-2349--regen-website.netlify.app/press-kit

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
